### PR TITLE
Correcting documenting for backend selection on the command line

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ A backend represents a way to run the user's command specified in the `task` sec
 * Platform Load Sharing Facility - Use `bsub` and job monitoring to run scripts.
 * Google JES - Launch jobs on Google Compute Engine through the Job Execution Service (JES).
 
-Backends are specified via the configuration option `backend.backend` which can accept the values: `sge`, `lsf`, `local`, and `jes` (e.g. `java -Dbackend.backend=sge`).
+Backends are specified via the configuration option `backend.defaultBackend` which can accept the values: `sge`, `lsf`, `local`, and `jes` (e.g. `java -Dbackend.defaultBackend=sge`).
```
Comment (Horneth) [inline README.md]: More like `The default backend is specified...` ?

**Comment by geoffjentry** [comment]
👍  assuming the information is correct ;)

**Comment by mcovarr** [comment]
:+1:

**Comment by kshakir** [comment]
(ditto jeff, including...) 👍

**Pull request merged**